### PR TITLE
Add `get_built()` to parse the Built field

### LINKED ISSUE #48: get_built() function that splits the Built field
With similar logic to https://github.com/wch/r-source/blob/89ec1150299f7be62b839d5d5eb46bd9a63653bd/src/library/tools/R/admin.R#L115-L128

```r
tools:::.split_description(tools:::.read_description("DESCRIPTION"))$Built
#> $R
#> [1] ‘3.4.1’
#> 
#> $Platform
#> [1] "x86_64-apple-darwin15.6.0"
#> 
#> $Date
#> [1] "2017-09-14 20:30:19 UTC"
#> 
#> $OStype
#> [1] "unix"
```

### PR DESCRIPTION
**Pull request by jimhester**
Fixes #48 

I don't know if we want to have additional functions for this field, I can add them if we do. Also not sure if we want to try and parse the date as a POSIXct object or not, I have left it as a character vector for now.

I added a Built field to one of the test DESCRIPTION files for testing, if you would prefer I created a new DESCRIPTION file instead I can do that.

### CODE REVIEW

File: R/description.R [hunk 1]
```diff
@@ -594,7 +603,13 @@ description <- R6Class("description",
       idesc_del_remotes(self, private, pattern),
 
     clear_remotes = function()
-      idesc_clear_remotes(self, private)
+      idesc_clear_remotes(self, private),
+
+    ## -----------------------------------------------------------------
+    ## Remotes
```
Comment (gaborcsardi) [inline R/description.R]: Why this header?

**Comment by gaborcsardi** [comment]
I think you forgot to add the `desc_get_built.Rd` file.

**Comment by jimhester** [comment]
Whoops! Should be there now.

**Comment by gaborcsardi** [comment]
Grat, thanks!

**Pull request merged**